BURN_BABY_BURN--MASTER_IGNITION_ROUTINE.s

```diff
@@ -55,7 +55,7 @@
 # BETWEEN THE PRE-IGNITION TIME CHECK -- ARE WE WITHIN 45 SECONDS OF TIG? -- AND TIG + 26 SECONDS, WHEN DPS
 # PROGRAMS THROTTLE UP.
 #
-# VARIATIONS AMONG PROGRAMS ARE ACCOMODATED BY MEANS OF TABLES CONTAINING CONSTANTS (FOR AVEGEXIT, FOR
+# VARIATIONS AMONG PROGRAMS ARE ACCOMMODATED BY MEANS OF TABLES CONTAINING CONSTANTS (FOR AVEGEXIT, FOR
```

Comment:
**Accomodated** is a typo present in [source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0731.jpg)